%YAML:1.0
# LoopClosureDetector parameters:
use_nss: 1
alpha: 0.001
min_temporal_matches: 1
dist_local: 10
max_db_results: 50
min_nss_factor: 0.05
min_matches_per_group: 1
max_intragroup_gap: 3
max_distance_between_groups: 3
max_distance_between_queries: 2

geom_check_id: 0
min_correspondences: 12

pose_recovery_option_id: 0
max_ransac_iterations_stereo: 500
ransac_probability_stereo: 0.995
ransac_threshold_stereo: 9
ransac_randomize_stereo: 1
ransac_inlier_threshold_stereo: 0.3
use_mono_rot: 0
relative_pose_info_damper: 0.9
lowe_ratio: 0.7
matcher_type: 4

nfeatures: 500
scale_factor: 1.2
nlevels: 8
edge_threshold: 31
first_level: 0
WTA_K: 2
score_type_id: 0
patch_sze: 31
fast_threshold: 20

pgo_uniform_weight: false
pgo_rot_threshold: 0.5
pgo_trans_threshold: 10

# geom_check_id options:
#   0: NISTER
#   1: NONE

# pose_recovery_option_id options:
#   0: RANSAC_ARUN
#   1: GIVEN_ROT

# matcher_type options:
#   1: FLANNBASED
#   2: BRUTEFORCE
#   3: BRUTEFORCE_L1
#   4: BRUTEFORCE_HAMMING
#   5: BRUTEFORCE_HAMMINGLUT
#   6: BRUTEFORCE_SL2
